src/includes/getting-started-install/apple.ios.mdx

```diff
@@ -1,12 +1,7 @@
 We recommend installing the SDK with CocoaPods. To integrate Sentry into your Xcode project, specify it in your _Podfile_:
 
 ```ruby {filename:Podfile}
-platform :ios, '8.0'
-use_frameworks! # This is important
-
-target 'YourApp' do
-  pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '{{ packages.version('sentry.cocoa') }}'
-end
+pod 'Sentry', '~> {{ packages.version('sentry.cocoa') }}'
```

Comment:
I'm unsure about this change. It seems, based on:

```
 use_frameworks! # This is important 
```

That because of folks not adding this, they had issues and we added this here because of that.

@HazAT could you please confirm?

Reply:
Yes, so back at the time you needed if you used Swift.
Also if you don't have this I think the way you import changes, so I would leave it unless you know 100% we don't need it anymore.

Reply:
Also another thing, the reason it was `pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '{{ packages.version('sentry.cocoa') }}'` before is that.
At one point Cocoapods said you need to define a source like `source 'https://github.com/CocoaPods/Specs.git'` with having `:git` in there it will fetch it from the repo which is better I think?

So yeah, `use_frameworks!` can be dropped potentially (if it's default already) but may we keep to longer version of installing the Pod like:

```
pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '{{ packages.version('sentry.cocoa') }}'
```